### PR TITLE
Group time tool

### PR DESCRIPTION
**Pull request by Brun04**
## Description
Group Time tool is a small dock which display a time series represented by a group of raster and/or vector layers.
* Display a time series group as a GIF or a video,
* Generate a GIF of the time series.

### CODE REVIEW

File: library/tools/group_time_tool/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+# Group Time Tool
+
+A LizMap custom JavaScript to display time series as a video or a GIF.
+<img style="float: right;" src="/images/timetool500.png" alt="Logo" width="20"/>
```
Comment (Gustry) [inline library/tools/group_time_tool/README.md:5]: Switch to markdown and it seems the link will lead to a 404.

File: library/tools/group_time_tool/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+			    |__ grouptimetool.js
+			    |__ timetool.html
+		    |__ gif.js-0.2.0/
+			    |__ gif.js
+			    |__ gif.js.map
+			    |__ gif.worker.js
+			    |__ gif.worker.js.map
+```
+
+For more details, consult the documentation available [here](/doc/Group_time_tool.pdf?inline=false)
```
Comment (Gustry) [inline library/tools/group_time_tool/README.md:34]: Same, this will lead to a 404

File: library/tools/group_time_tool/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Example
+
+![](/doc/covid19-Deaths_FR.gif)
```
Comment (Gustry) [inline library/tools/group_time_tool/README.md:38]: 404 as well

**Review by Gustry — COMMENTED** [review]
I think you should use absolute links from the root repository.
Nice readme with GIF !

**Comment by Brun04** [comment]
Thanks for the feedback. Normally, links should work with the last commit.

File: library/tools/group_time_tool/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+
+# Group Time Tool
+
+A LizMap custom JavaScript to display time series as a video or a GIF.
+<img src="https://github.com/Brun04/lizmap-javascript-scripts/blob/master/library/tools/group_time_tool/images/timetool500.png" width=20px style="float: right;">
```
Comment (Gustry) [inline library/tools/group_time_tool/README.md:5]:
```suggestion
![](./images/timetool500.png)
```

File: library/tools/group_time_tool/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+			    |__ grouptimetool.js
+			    |__ timetool.html
+		    |__ gif.js-0.2.0/
+			    |__ gif.js
+			    |__ gif.js.map
+			    |__ gif.worker.js
+			    |__ gif.worker.js.map
+```
+
+For more details, consult the documentation available [here](https://github.com/Brun04/lizmap-javascript-scripts/blob/master/library/tools/group_time_tool/doc/Group_time_tool.pdf?inline=false)
```
Comment (Gustry) [inline library/tools/group_time_tool/README.md:35]:
```suggestion
For more details, consult the documentation available [here](./doc/Group_time_tool.pdf)
```

File: library/tools/group_time_tool/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+## Example
+
+![](https://github.com/Brun04/lizmap-javascript-scripts/blob/master/library/tools/group_time_tool/doc/covid19-Deaths_FR.gif)
```
Comment (Gustry) [inline library/tools/group_time_tool/README.md:39]:
```suggestion
![](./doc/covid19-Deaths_FR.gif)
```

**Review by Gustry — COMMENTED** [review]
Sorry, I think it should be OK like this

**Comment by Brun04** [comment]
Indeed it's better in that way

**Review by Gustry — APPROVED** [review]

**Comment by Gustry** [comment]
Thanks for your contribution !

Sorry for the long delay !

**Pull request merged**